chain-core/src/init/params.rs

```diff
@@ -1,3 +1,4 @@
+use super::config::DistributionError;
```

Comment:
Some options:

1. return `Result<(), ()>` in validation here
2. validation + import conditioned by `#[cfg(not(feature = "mesalock_sgx"))]`
3. move this validation to config

config module isn't compiled in sgx, because it's not needed for the existing functionality + it brings extra dependencies (base64, serde, serge-json,. ...)

Reply:
I think option 3 is simply enough.

Reply:
Also, there are some out of date network parameters and namings, we can solve them in another issue.

Reply:
do you mean this: https://github.com/crypto-com/chain/issues/874 ?

Reply:
> do you mean this: #874 ?

I mean the `jailing_config` has nothing to do with jail now, it seems that's the only thing out of date.